demos/common/python/models/deblurring.py

```diff
@@ -20,15 +20,16 @@
 class Deblurring(Model):
     def __init__(self, ie, model_path, input_image_shape):
         super().__init__(ie, model_path)
-        self.calculate_new_shape(input_image_shape)
+        self.block_size = 32
+        self.input_shape = input_image_shape
+        self.calculate_new_shape()
         self.input_blob_name = self.prepare_inputs()
         self.output_blob_name = self.prepare_outputs()
 
-    def calculate_new_shape(self, shape):
-        h, w, _ = shape
-        block_size = 32
-        new_height = math.ceil(h / block_size) * block_size
-        new_width = math.ceil(w / block_size) * block_size
+    def calculate_new_shape(self):
+        h, w, _ = self.input_shape
```

Comment:
1. `Deblurring` shouldn't store `input_image_shape`'s third value because the last value isn't used.
2. `postprocessing()` is meant to revert what was done in `preprocessing()` in terms of resize. `postprocessing()` should resize image back to original size, which you've stored for yourself in `preprocessing()` in `meta`. That means you don't need `self.input_shape` field at all: `meta` should be used instead. BTW, I just noticed, it would be great if you cleaned `meta`: `resized_image.shape` isn't used as well as `image.shape[2]`.